docs/faq_build.md

```diff
@@ -49,9 +49,9 @@ Re-running the QMK installation script (`./util/qmk_install.sh` from the `qmk_fi
 If that doesn't work, then you may need to download and run Zadig. See [Bootloader Driver Installation with Zadig](driver_installation_zadig.md) for more detailed information.
 
 ## USB VID and PID
-You can use any ID you want with editing `config.h`. Using any presumably unused ID will be no problem in fact except for very low chance of collision with other product.
+You can use any ID you want with editing `config.h`. Using an ID that is presumed to be unused should generally not be a problem, except for the possibility of a rare collision with another product. A unique PID/VID combination is not required, but it is recommended.
```

Comment:
Might be worth noting that the 0x3000 block is being actively assigned, and getting pretty far in the range.  If you're avoiding collisions, the further past 0x4000, the better.

Reply:
@drashna 
I updated the docs with your note about avoiding collisions above 0x4000. 
I'm a bit confused though - I thought the VID/PID codes correspond to integers in hexadecimal format and not memory blocks - so there wouldn't necessarily be ranges? Maybe I'm missing something?

I first encountered the VID/PID when trying to setup VIA, and my first reference there was the [scotto keeb](https://www.youtube.com/watch?v=7d5yzBOup9U&themeRefresh=1) video, which presents the VID/PID as 2 characters converted to hexadecimal. I suspect many other folks will start there too.

What would be the best way to approach the VID/PID conceptually for people new to it? Maybe something like:
> Vendor ID (VID) and Product ID (PID) codes are unique identifiers assigned to USB devices by the USB Implementers Forum (USB-IF) to distinguish them from other USB devices. These codes are used by the USB host to identify the type and manufacturer of the USB device. VID and PID are always represented in hexadecimal,  prefix  written in hexadecimal format with a "0x". QMK does not require a unique PID/VID combination.

Reply:
The IDs are arbitrary and don't actually represent anything.  But the USB-IF (eg, the regulatory body that controls the USB specs, vendor ID assignments, etc) hand them out in a sequential manner.  (there are gaps and such in lower values, but for the 0x3xxx block, it's been very sequential).  

Anything like scotto's stuff is just fluff around that, and an easy way to pick a VID/PID. I've seen sha hashs of the keyboard/manu's name being used for generating it too.  

So really, there isn't any real association. It's just a number that is assigned, and used to identify the hardware.